css/types/ratio.json

```
@@ -48,6 +48,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "as_number": {
```

This is mostly a personal thought, but maybe we should make the data point's name `number_value`?

```suggestion
        "number_value": {
```